src/services/cli-http-client.js

```diff
@@ -93,13 +93,16 @@ class CliRequestClient {
       this.logger.debug(`response.statusCode: ${response.status}`);
       this.logger.debug(`response.headers: ${JSON.stringify(response.headers)}`);
 
+      /* eslint-disable camelcase */
       if (response.status < 200 || response.status >= 400) {
-        const parsed = response.data;
+        const { code, message, more_info, details } = response.data;
```

Comment:
Does `response.data` always include these four return values? What happens if one is missing. (e.g. we have `details`, but not `more_info`).

Reply:
I believe it will always return code, message, and more_info as that logic hasn't changed, but, in the worst case scenario, it would return `undefined` for those values (it would have done this before anyway if I'm not mistaken). 

However, I could add a default value to the parameters to make it look nicer in the event of an `undefined` value. For example, instead of seeing `details: undefined` in the output, I could set it to default to a string like `details: "No details provided"` and do that for the other properties.

Reply:
I like the default values idea. Thanks!